numpy/core/tests/test_arrayprint.py

```diff
@@ -199,8 +198,7 @@ def test_unstructured_void_repr(self):
         assert_equal(str(a[0]), r"b'\x1B\x5B\x32\x4B\x07\x41\x0A\x08'")
         assert_equal(repr(a),
             r"array([b'\x1B\x5B\x32\x4B\x07\x41\x0A\x08'," "\n"
-            r"       b'\x1B\x5B\x33\x31\x6D\x52\x65\x64']," "\n"
-            r"      dtype='|V8')")
+            r"       b'\x1B\x5B\x33\x31\x6D\x52\x65\x64'], dtype='|V8')")
```

Comment:
Sicne you touched it anyway, this test might be a lot clearer with `textwrap.dedent("""\` ... `""")` to handle the multiline indented string

Reply:
Yeah I didn't know about that until I saw you use it in the maskedarray PR. Good idea!

Reply:
I think this line is better without dedent because it uses the `r` raw mode, which prevents me from starting the string with `"""\`.

The lines below are better with dedent, I'll change those.

Reply:
Raw should work with `"""`, some of the docstrings are raw strings.

Reply:
The problem I'm having is that `textwrap.dedent` needs the starting newline to be escaped, ie the arg should start with `"""\` (escaped newline). But in a raw string I can't escape the newline.

For instance the following script:
```python
from __future__ import print_function
import textwrap
print(textwrap.dedent(r"""\
    aaa
    bbb
    ccc"""))
```
prints
```
\
    aaa
    bbb
    ccc
```
(curiously the ipython shell seems to correctly escape the newline, but not the plain python shell nor scripts)

Reply:
But anyway, I can fix it by just avoiding the newline... will fix.

Reply:
You're right, I hadn't though about the raw string interfering with the initial newline, Thanks for pointing that out to me before I run into it elsewhere!